libs/datamodel/core/tests/attributes/relations/referential_actions.rs

```diff
@@ -399,19 +399,17 @@ fn restrict_should_not_work_on_sql_server() {
     "#};
 
     let expected = expect![[r#"
-        [1;91merror[0m: [1mError parsing attribute "@relation": Invalid referential action: `Restrict`. Allowed values: (`Cascade`, `NoAction`, `SetNull`, `SetDefault`)[0m
+        [1;91merror[0m: [1mError validating: Invalid referential action: `Restrict`. Allowed values: (`Cascade`, `NoAction`, `SetNull`, `SetDefault`)[0m
```

Comment:
Wdyt about `Error validating relation` or maybe `Validation error`?

Reply:
Should we change the error functions?

Reply:
Long term yes, for now we can ignore too

Reply:
yup